app/controllers/application_controller.rb

```diff
@@ -11,6 +11,13 @@ def ps
 
   private
 
+  helper_method def navbar_hide?
```

Comment:
WDYT if we just created separate layouts and tell these controllers to use them?

Reply:
! Thank you